src/system.rs

```diff
@@ -185,6 +194,13 @@ pub trait System<'a> {
     fn setup(&mut self, world: &mut World) {
         <Self::SystemData as DynamicSystemData>::setup(&self.accessor(), world)
     }
+
+    /// Performs clean up that requires resources from the `World`.
+    /// This commonly removes components from `world` which depend on external
+    /// resources.
+    fn dispose(&mut self, world: &mut World) {
+        let _ = world;
```

Comment:
do you actually need to do this?

Reply:
If I want no warning and keep `world` (instead of `_world`) as a parameter, yes :)

Reply:
why not just `_world`?

Reply:
It shows up in the docs ^^

Reply:
but blanket impls you dont care about the fn's generally you just care about that the bounds for T work for you no? I mean this is minor i dont mind either way 😅

Reply:
This is not a blanket impl, it's the `System` trait. I just wanted to have a nice parameter show up in the docs :sweat_smile:

Reply:
Oh my god! You're right I missed that haha, I guess I didnt expand the diff hahaha. you're right this works! Sorry!